app/uk/gov/hmrc/servicedependencies/model/Team.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.servicedependencies.model
+
+import play.api.libs.json.{Format, JsError, JsResult, JsSuccess, JsValue, Json, Writes}
+
+case class Team(
+                 name: String,
+                 repos: Map[String, Seq[String]]
+               ) {
+  def allRepos: Seq[String] =
+    repos.values.toSeq.flatten
+
+  private def findRepo(name: String): Seq[String] = repos.getOrElse(name, Seq.empty)
+
+  def services   : Seq[String] = findRepo("Service")
+  def libraries  : Seq[String] = findRepo("Library")
+  def others     : Seq[String] = findRepo("Other")
+  def prototypes : Seq[String] = findRepo("Prototype")
+
+}
+
+object Team {
+  case class ApiTeam(name: String,
```

Comment:
It looks like writing an appropriate format explicitly would make the ApiTeam object unnecessary.